tests/Offer/DefaultOfferEditingServiceTest.php

```diff
@@ -36,6 +36,11 @@ class DefaultOfferEditingServiceTest extends TestCase
      */
     private $offerRepository;
 
+    /**
+     * @var DocumentRepository|MockObject
+     */
+    private $organizerDocumentRepository;
```

Comment:
Also no need here for the property. I see that this is done with several mocks in this class.